lib/packages/core/src/api/factories/account/setAccountInfo.ts

```diff
@@ -0,0 +1,49 @@
+/** @ignore */
+/** @module core */
+
+/**
+ * Copyright (c) 2019 Burst Apps Team
+ */
+import { BurstService } from '../../../burstService';
+import { TransactionId } from '../../../typings/transactionId';
+import { TransactionResponse } from '../../../typings/transactionResponse';
+import { generateSignature, decryptAES, Keys } from '@burstjs/crypto';
+import { verifySignature } from '@burstjs/crypto';
+import { generateSignedTransactionBytes } from '@burstjs/crypto';
+import { convertNumberToNQTString } from '@burstjs/util';
+import { broadcastTransaction } from '../transaction/broadcastTransaction';
+
+export const setAccountInfo = (service: BurstService): (
+        name: string,
+        description: string,
+        feeNQT: string,
+        senderPublicKey: string,
+        senderPrivateKey: string,
+        deadline: number,
+    ) => Promise<TransactionId | Error> =>
+    async (
+        name: string,
+        description: string,
+        feeNQT: string,
+        senderPublicKey: string,
+        senderPrivateKey: string,
+        deadline: number,
+    ): Promise<TransactionId | Error> => {
+
+        let parameters = {
+            name,
+            description,
+            deadline: 1440,
```

Comment:
todo: use the deadline provided by the client

Reply:
what does the deadline mean in this context...?

Reply:
From the docs: `deadline deadline (in minutes) for the transaction to be confirmed, 1440 minutes maximum`